src/current/cockroachcloud/network-authorization.md

```diff
@@ -41,10 +48,20 @@ Authorized network access can be managed from the {{ site.data.products.db }} co
 
 `https://cockroachlabs.cloud/cluster/{ your cluster UUID}/networking`
 
-You can add up to 20 IP addresses to your allowlist. If your application servers’ IP addresses are not static, or you want to limit your cluster's exposure to the public network, you can use VPC Peering or AWS PrivateLink instead (Dedicated clusters only).
+Serverless and Dedicated clusters support different maximum number of IP allowlist rules/authorized sub-networks:
```

Comment:
Maybe an "a" before "different" here? "... support a different maximum number of IP..."